components/common-data-items.js

```diff
@@ -257,18 +257,15 @@ export function SampleDataItems({
       {item.protocols?.length > 0 && (
         <>
           <DataItemLabel>Protocols</DataItemLabel>
-          <DataItemList isCollapsible isUrlList>
-            {item.protocols.map((protocol) => (
-              <a
-                href={protocol}
-                key={protocol}
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                {protocol}
-              </a>
-            ))}
-          </DataItemList>
+          <DataItemValue>
+            <SeparatedList>
+              {item.protocols.map((protocol) => (
+                <a href={protocol} key={protocol}>
```

Comment:
To be consistent, open the protocol page in a new tab, so you can use the `target` and `rel` attributes you had.

Reply:
done